packages/testkit-backend/src/skipped-tests/browser.js

```diff
@@ -1,8 +1,16 @@
-import skip, { ifStartsWith } from './skip'
+import skip, { ifEndsWith, ifEquals, ifStartsWith } from './skip'
 const skippedTests = [
   skip(
-    'Stub Tests not implemented for browser',
-    ifStartsWith('stub')
+    'Browser doesn\'t support socket timeouts',
```

Comment:
It's your code-base and I'm merely curious if this is preferable over
```suggestion
    "Browser doesn't support socket timeouts",
```